packages/smooth_app/lib/pages/hunger_games/question_page.dart

```diff
@@ -18,12 +18,14 @@ import 'package:smooth_app/widgets/smooth_scaffold.dart';
 
 class QuestionPage extends StatefulWidget {
   const QuestionPage({
+    required this.shouldDisplayContinueButton,
```

Comment:
Looks like `final bool shouldDisplayContinueButton = product != null;`, right?
If so, we don't need a parameter or a field, just a getter.

Reply:
Okay, Will make necessary changes and push.